packages/www/lib/utils/index.tsx

```diff
@@ -109,6 +109,23 @@ export const getStripe = () => {
   return stripePromise;
 };
 
+/**
+ * should stripe be enabled in this context?
+ */
+export const shouldStripe = () => {
```

Comment:
would be really funny if someone typo'd the `e` out 🌚

Reply:
lmao